test/nginx_system_test.sh

```diff
@@ -523,11 +528,24 @@ check test $(scrape_stat image_rewrite_total_original_bytes) -ge 10000
 # happens both before and after.
 start_test "Reload config"
 
+AB_PID="0"
+# Fire up some heavy load if ab is available to test a stressed shutdown
+if hash ab 2>/dev/null; then
+    ab -n 10000 -c 100 -k http://$PRIMARY_HOSTNAME/ &>/dev/null & AB_PID=$!
+    # Sleep to allow some queueing up of requests
+    sleep 2
+fi
```

Comment:
If `ab` isn't present, could we print something about being sad we can't run this test?